src/components/ProductCard/ProductCard.tsx

```diff
@@ -2,13 +2,13 @@ import { useState } from 'react';
 import styles from './styles.module.scss';
 import blankIcon from '../../assets/images/icons/favorites-blank.svg';
 import filledIcon from '../../assets/images/icons/favorites-filled.svg';
-import { PrimaryButtons } from '../../enums';
 import { useAddCartButton, useRemoveFromCartButton } from '../../hooks';
 import { FavoritesButton, PrimaryButton } from '../index';
 import { useSelector } from 'react-redux';
 import { RootState } from '@/app/store';
 import { toggleClickedBuy } from '@/features/сart/сartSlice';
 import { Device } from '@/types/Device';
+import { PrimaryButtons } from '@/enums/PrimaryButtons';
```

Comment:
```suggestion
import { PrimaryButtons } from '@/enums';
```

There's an index file in there, so it should work that way. Also pay attention to the order of imports.